web/modules/custom/hoeringsportal_audit_log/src/Helpers/ConfigHelper.php

```diff
@@ -124,20 +147,26 @@ public function isConfigActive(string $currentRouteName, string $entityTypeId, ?
     $types = $this->getConfiguration('types');
 
     // If no types configuration exists, return an empty array immediately.
-    if (!$types) {
+    if (!$types && is_array($types)) {
       return FALSE;
     }
 
     // Only some entity types has a bundletype, if they do, we need that to find
     // it in config if we dont, the configuration just has the entity type
     // twice. (e.g. user -> user)
     $typeId = $bundleType ?: $entityTypeId;
+    $escapedRoute = $this->escapeProviderId($currentRouteName);
+
+    $type = $types[$entityTypeId] ?? NULL;
 
-    $escapedRouteName = $this->escapeProviderId($currentRouteName);
+    if (is_array($type) && is_array($type[$typeId])) {
```

Comment:
This will report an error if `$typeId` is not set in `$type`:

```suggestion
    if (isset($type[$typeId]) && is_array($type[$typeId])) {
```